.changelog/2758.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+core/server: add new grpc endpoint UpserWorkspace
```

Comment:
don't think we need a changelog entry since our API isn't documented atm

Reply:
👍 removed in d9f7fc7f4